index.test.js

```diff
@@ -0,0 +1,31 @@
+var expect = require('chai').expect;
+var countdown = require('./index');
+var moment = require('moment');
+
+describe("countdown message", function() {
+  it("is a string", function() {
```

Comment:
For so many assertions, and multiple evaluations of countdown.composeMessage, what do you think about splitting this into multiple tests? This is doing much more than checking if the return value is a string.